support/kueue.go

```diff
@@ -65,7 +64,25 @@ func CreateKueueClusterQueue(t Test, clusterQueueSpec kueuev1beta1.ClusterQueueS
 	return clusterQueue
 }
 
-func CreateKueueLocalQueue(t Test, namespace, clusterQueueName string) *kueuev1beta1.LocalQueue {
+type DefaultLocalQueueOption struct {
+	namespace        string
+	clusterQueueName string
+	annotations      map[string]string
```

Comment:
I don't think these are needed. That option is only to apply the default queue annotation.